src/models/bmdashboard/buildingInventoryType.js

```diff
@@ -59,8 +59,23 @@ const reusableType = invTypeBase.discriminator('reusable_type', new mongoose.Sch
 
 const toolType = invTypeBase.discriminator('tool_type', new mongoose.Schema({
   category: { type: String, enum: ['Tool'] },
-  isPowered: { type: Boolean, required: true },
-  powerSource: { type: String, required: () => this.isPowered }, // required if isPowered = true (syntax?)
+  invoice: String,
+  purchaseRental: String,
+  fromDate: Date,
+  toDate:Date,
+  condition: String,
+  phoneNumber: String,
+  quantity: Number,
+  currency: String,
+  unitPrice: Number,
+  shippingFee: Number, 
+  taxes: Number, 
+  totalPriceWithShipping:  Number,
+  images: String,
+  link: String,
+
+  // isPowered: { type: Boolean, required: true },
+  // powerSource: { type: String, required: () => this.isPowered }, // required if isPowered = true (syntax?)
```

Comment:
Hi Shereen,
Can we get rid of comments on line 77 & 78 if they do not serve any useful purpose?